app/docs/src/reference/management.md

```diff
@@ -0,0 +1,352 @@
+            resourceId?: string | null;
+          } | null;
+          domain?: {
+            // Properties based on the type of the component being managed
+            // ...
+          } | null;
+          secrets?: {
+            credential?: string | null;
+          } | null;
+          resource?: {
+            status?: "ok" | "warning" | "error" | undefined | null;
+            message?: string | null;
+            payload?: any | null;
+            last_synced?: string | null;
+          } | null;
+          resource_value?: {} | null;
+          code?:
+            | Record<
+              string,
+              {
+                code?: string | null;
+                format?: string | null;
+              }
+            >
+            | null;
+          qualification?:
+            | Record<
+              string,
+              {
+                result?: string | null;
+                message?: string | null;
+              }
+            >
+            | null;
+          deleted_at?: string | null;
+        };
+        geometry?: Geometry;
+        connect?: {
+          from: string;
+          to: {
+            component: string;
+            socket: string;
+          };
+        }[];
+        parent?: string;
+      };
+    };
+    update?: {
+      [key: string]: {
+        properties?: { [key: string]: unknown };
+        geometry?: { [key: string]: Geometry };
+        connect?: {
+          add?: { from: string; to: { component: string; socket: string } }[];
+          remove?: {
+            from: string;
+            to: { component: string; socket: string };
+          }[];
+        };
+        parent?: string;
+      };
+    };
+    actions?: {
+      [key: string]: {
+        add?: ("create" | "update" | "refresh" | "delete" | string)[];
+        remove?: ("create" | "update" | "refresh" | "delete" | string)[];
+      };
+    };
+  };
+  message?: string | null;
+};
+```
+
+The `geometry` properties are deprecated and no longer used.
+
+## Common Use Cases
+
+Management functions are commonly used for:
+
+### Importing Existing Resources
+
+Import functions discover resources that already exist in your cloud provider
+and create corresponding components in System Initiative with their current
+configuration.
+
+For example, an AWS VPC import function would:
+
+1. Query AWS for VPC details using a resource ID
+2. Create or update a VPC component with the discovered configuration
+3. Switch the component from "create" to "refresh" actions
+4. Return the imported configuration
+
+### Creating Templates
+
+Template functions generate complete infrastructure patterns by creating
+multiple related components with proper connections and configuration.
+
+For example, a "VPC Template" function might:
+
+1. Create a VPC component
+2. Create public and private subnets as children
+3. Create route tables and make subscriptions to them
+4. Create an internet gateway and make subscriptions to it
```

Comment:
Templates are no longer supported in this manner; at least not officially.

Am I right here, @stack72?